lib/kb_phylogenomics/kb_phylogenomicsImpl.py

```diff
@@ -1060,7 +1060,8 @@ def view_fxn_profile(self, ctx, params):
                         #if f_cnt % 100 == 0:
                         #    self.log (console, "prot: "+str(feature['protein_translation']))  # DEBUG
 
-                        if 'function' in feature and feature['function'] != None and feature['function'] != '':
+                        if ('function' in feature and feature['function'] != None and feature['function'] != '') \
+                           or ('functions' in feature and feature['functions'] != None and len(feature['functions']) != 0):
```

Comment:
You can write this line more concisely as if feature.get('function') or feature.get('functions').

.get() returns None for missing keys (or provide a second argument to supply the default value) and None, "", and [] are falsey